docs/tutorial/tutorial_orbit.rst

```diff
@@ -25,154 +29,199 @@ orbit periods aren't constant, a 100% success rate is not be guaranteed.
+   # To ensure we have a complete first orbit
+   f15.load(date=day2)
+   print(f15.index[0], f15.index[-1])
 
-.. code:: ipython
+   2012-01-01 00:02:09 2012-01-02 00:00:01
 
-   ivm.orbits[1]
 
-   Returning cnofs ivm  data for 12/27/12
-   Returning cnofs ivm  data for 12/28/12
-   Loaded Orbit:0
-
-Note that getting the first orbit caused pysat to load the day previous, and
-then back to the current day. Orbits are zero indexed.
-pysat is checking here if the first orbit for 12/28/2012 actually started on
-12/27/2012. In this case it does.
+Orbits may be selected directly from the attached ``f15.orbits`` class.
+The data for the orbit is stored in ``f15.data``.
 
 .. code:: ipython
 
-   ivm[0:5, 'mlt']
+   f15.orbits[1]
+   print(f15.index[0], f15.index[-1])
 
-   2012-12-27 23:05:14.584000    0.002449
-   2012-12-27 23:05:15.584000    0.006380
-   2012-12-27 23:05:16.584000    0.010313
-   2012-12-27 23:05:17.584000    0.014245
-   2012-12-27 23:05:18.584000    0.018178
-   Name: mlt, dtype: float32
+   2012-01-01 00:12:21 2012-01-01 01:03:09
 
-   ivm[-5:, 'mlt']
 
-   2012-12-28 00:41:50.563000    23.985415
-   2012-12-28 00:41:51.563000    23.989031
-   2012-12-28 00:41:52.563000    23.992649
-   2012-12-28 00:41:53.563000    23.996267
-   2012-12-28 00:41:54.563000    23.999886
-   Name: mlt, dtype: float32
+Note that getting the first orbit caused pysat to load the day previous, and
+then back to the current day. This is why the start of the day and the start
+of the first orbit do not coincide.
 
-You can also go back an orbit.
+You can go forward an orbit, to see what a full orbit looks like:
 
 .. code:: ipython
 
-   ivm.orbits.prev()
-
-   Returning cnofs ivm  data for 12/27/12
-   Loaded Orbit:15
-
-   ivm[-5:, 'mlt']
+   f15.orbits.next()
+   print(f15.index[0], f15.index[-1])
 
-   2012-12-27 23:05:09.584000    23.982796
-   2012-12-27 23:05:10.584000    23.986725
-   2012-12-27 23:05:11.584000    23.990656
-   2012-12-27 23:05:12.584000    23.994587
-   2012-12-27 23:05:13.584000    23.998516
-   Name: mlt, dtype: float32
+   2012-01-01 01:03:13 2012-01-01 01:54:01
 
-pysat loads the previous day, as needed, and returns the last orbit for
-12/27/2012 that does not (or should not) extend into 12/28.
-
-If we continue to iterate orbits using
-
-.. code:: python
-
-   ivm.orbits.next()
-
-eventually the next day will be loaded to try and form a complete orbit. You
-can skip the iteration and just go for the last orbit of a day,
+And back an orbit:
 
 .. code:: ipython
 
-   ivm.orbits[-1]
+   f15.orbits.prev()
+   print(f15.index[0], f15.index[-1])
 
-   Returning cnofs ivm  data for 12/29/12
-   Loaded Orbit:1
+   2012-01-01 00:12:21 2012-01-01 01:03:09
 
-.. code:: ipython
+If we continue to iterate orbits using ``f15.orbits.next()`` the next day will
+eventually be loaded to try and form a complete orbit. You can skip the
+iteration and just go for the last orbit of a day using indexing:
```

Comment:
With the above download routines, this will not load the next day of data for new users.  Should be noted or fixed above.

Reply:
No, it does.  I didn't have these files or the times around them and downloaded them using the above command.  Did it not work for you?